docs/src/userguide/citation.rst

```diff
@@ -15,11 +15,12 @@ For example::
 
  @manual{Iris,
  author = {{Met Office}},
- title = {Iris: A Python package for analysing and visualising meteorological and oceanographic data sets},
- edition = {v1.2},
- year = {2010 - 2013},
+ title = {Iris: A powerful, format-agnostic, and community-driven Python package for analysing and visualising Earth science data },
+ edition = {v3.4},
+ year = {2010 - 2022},
  address = {Exeter, Devon },
- url = {http://scitools.org.uk/}
+ url = {http://scitools.org.uk/},
+ doi = {10.5281/zenodo.595182}
```

Comment:
```suggestion
 doi = {10.5281/zenodo.7386117}
```
is the DOI for iris 3.4 (see https://doi.org/10.5281/zenodo.595182)

Reply:
`595182` is the parent ID for all releases of Iris, and automatically redirects to the latest release. I prefer `595182` as it avoids a manual update for each new release.

Reply:
I did choose `595182` so that it was valid for all versions. If someone explicitly wants to reference a specific version then they can update the DOI.

Reply:
At the moment, it looks like this BibTex file should be considered an example. It refers to a specific version of Iris on line 19 and a specific year on line 20. Just replacing the version-specific DOI with the Concept DOI won't prevent it from going out of date.

Reply:
I'm convinced, @bouweandela's suggestion looks like the way to go!